doc/nrf/external_comp/bt_fast_pair.rst

```diff
@@ -888,6 +888,16 @@ The specific use case of the Google Fast Pair application is indicated by the ch
 In the official `Fast Pair`_ documentation, the `Fast Pair Device Feature Requirements`_ category defines additional requirements for each supported use case, and specifies a list of mandatory, optional, and unsupported Fast Pair features.
 If your product is targeting one of the listed use cases, you must align your accessory firmware to meet these requirements.
 
+If your Google Fast Pair use case is supported by the |NCS|, you can select it using the :kconfig:option:`CONFIG_BT_FAST_PAIR_USE_CASE` Kconfig choice option.
```

Comment:
This could say something along the lines: ``Select an option depending on your use case:``
I'd also say agree that the doc structure makes listing these references unnecessary, but at the same time the article as a whole is pretty overwhelming, so it definitely wouldn't hurt to help user navigate